src/raftstore/store/peer.rs

```diff
@@ -911,6 +910,15 @@ impl Peer {
                         "peer_id" => self.peer.get_id(),
                         "lease" => ?self.leader_lease,
                     );
+                    // If predecessor read index during transferring leader and received quorum's
+                    // heartbeat response, it may waiting for advancing apply to current term to
```

Comment:
it may wait

Reply:
> waiting for advancing apply to current term to apply the read.
What does "advancing apply to current term" mean?  @BusyJay